commit fce4317b35a37bf0b8dbe416f28a73bac9abf105 (refs/changes/32/5632/2)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2019-02-25T18:39:14+00:00 (8 months ago)
    
    TRITON-1251 vmapi flexible_disk_size tests fail on mockcloud CNs
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/test/vms.disks.test.js b/test/vms.disks.test.js
index aa732f3..80e9786 100644
--- a/test/vms.disks.test.js
+++ b/test/vms.disks.test.js
@@ -35,7 +35,10 @@ var VM_OPTS = {
     ram: 1024,
     disks: [ { image_uuid: '81c0ef69-e9d7-4e93-a15b-efd7ea9c9ee8' } ],
     networks: [],
-    creator_uuid: CUSTOMER_UUID
+    creator_uuid: CUSTOMER_UUID,
+    tags: {
+        'triton.placement.exclude_virtual_servers': true
+    }
 };
 var CALLER = {
     type: 'signature',
